src/nanodbc/nanodbc.cpp

```diff
@@ -102,6 +102,16 @@
 #endif
 #endif
 
+#if defined(NANODBC_OVERALLOCATE_CHAR)
```

Comment:
Are we diverging from nanodbc here, or did you get this merged upstream?

Reply:
Yep, [merged upstream.](https://github.com/nanodbc/nanodbc/pull/380)

Reply:
Nice, thanks for all your work on this!